.env.template

```diff
@@ -94,9 +94,15 @@ OPENAI_API_KEY=your-openai-api-key
 
 ### MILVUS
 ## MILVUS_ADDR - Milvus remote address (e.g. localhost:19530)
-## MILVUS_COLLECTION - Milvus collection,
-## change it if you want to start a new memory and retain the old memory.
-# MILVUS_ADDR=your-milvus-cluster-host-port
+## MILVUS_URI - Zilliz public endpoint (e.g. https://xxx-xxxx.xxxx.xxxx.zillizcloud.com:443)
+##   If set, MILVUS_URI overrides MILVUS_ADDR.
+## MILVUS_USERNAME - username of cloud service or milvus
+## MILVUS_PASSWORD - password of cloud service or milvus
+## MILVUS_COLLECTION - Milvus collection, change it if you want to start a new memory and retain the old memory.
+# MILVUS_ADDR=localhost:19530
```

Comment:
It seems that one configuration is enough for Milvus address/URI， and enables secure options if the URI contains https prefix before creating the connection.

Reply:
I merge the URI and ADDR in new commit. it looks better now.